### DISCUSSION TITLE
Code Linting not working

### DISCUSSION THREAD

**Discussion by dipeshyadav331** [cpeditor/cpeditor#1237] [Q&A]
I was setting up code linting in java in cp editor.

In the documentation , this file is given [org.eclipse.equinox.launcher_1.5.200.v20180922-1751.jar](https://cpeditor.org/docs/setup/#:~:text=The%20full%20name%20of%20the%20jar%20file%20above%2C%20org.eclipse.equinox.launcher_1.5.200.v20180922%2D1751.jar) , I found this file and added this to the LSP executable file  ( /Applications/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.6.400.v20210924-0641.jar ) 

and also I added the arguments as mentioned in the documents
(--configuration /Applications/Eclipse.app/config_mac)

but still I am getting this error every time

[04:48:50] [Language Server [Java]] [Failed to start LSP Process. Have you set the path to the Language Server program at [Preferences->Extensions->Language Server->Java Server->Path to LSP executable](https://github.com/cpeditor/cpeditor/discussions/new?category=q-a#Preferences/Extensions/Language%20Server/Java%20Server/Path%20to%20LSP%20executable)?]

I am not understanding how to fix this , I have setup cf tool , code formatter using clang but even after spending a lot of time I am unable to make code linting work in cp editor. It is working fine in eclipse though.

can you please help me.
@ouuan

**Comment by dipeshyadav331** [comment]
@ouuan

**Comment by ouuan** [comment]
I don't use Java and I don't use macOS. But I think the executable file should be `java`, and the jar file should be an argument after `-jar`?
